package.json

```diff
@@ -17,5 +17,8 @@
   },
   "scripts": {
     "install": "prades install"
+  },
+  "dependencies": {
+    "@sb/prades": "^0.4.0"
```

Comment:
I don't think it is a good idea that have prades a dependency of the package itself. 
I understand there is a general lack of configuration management, but tools such as package manager, compilateurs ... should be managed globally with package management and not managed at each package level. 
It's obviously an incomplete temporary fix to manage the presence of this tool here, because it does not manage the other tools needed. But it also makes things really complex to manage when it is decided node should be upgraded, and therefore prades should to, because all the packages with such tool dependency (who maintains that list?) will have to be revisited and republished!!  Which is never going to happen in a million years. 

Also, this has the issue to install the same tool - in potentatially different versions, as many times as package, not counting the global one installed. 

Also, note that if someone has a prades installed globally, it make take over the one references in this package, so the version referenced here may not be the one used by the client anyways. 

So I am asking this to be removed from this package because it's going to polute all users of such package, and it does not remove the necessity to install prades globally to take care of packages that do not include this dependency